app/controllers/supplejack_api/admin/sessions_controller.rb

```diff
@@ -16,7 +16,7 @@ class SessionsController < Devise::SessionsController
       layout 'supplejack_api/application'
 
       def after_sign_in_path_for(resource)
-        stored_location_for(resource) || admin_users_path
+        stored_location_for(resource) || admin_site_activities_path
```

Comment:
What else do we need to sign in for on the api?  Can we delete this method altogether?

Reply:
There is one other part that has site activities. Once that has been removed we can get rid of all the views :D.